mysql-test/include/rpl_change_topology.inc

```diff
@@ -225,7 +225,14 @@ if (!$rpl_skip_change_master)
       {
         --let $_rpl_master_log_pos=
       }
```

Comment:
I'd treat explicitly supplied `master_log_file` as implicit `master_use_gtid = NO`. And this, according to the first of two if:s seems to be in place.